Backend/Remora.Discord.API.Abstractions/API/Objects/Activities/IActivity.cs

```diff
@@ -51,7 +51,7 @@ public interface IActivity
         /// <summary>
         /// Gets the time when the activity was added to the user's session.
         /// </summary>
-        Optional<DateTimeOffset> CreatedAt { get; }
+        DateTimeOffset CreatedAt { get; }
```

Comment:
This property is actually optional; the Discord docs are just somewhat confusing. See the note about bots only being able to send `name`, `type`, and `url`.

https://discord.com/developers/docs/topics/gateway#activity-object

Reply:
The entire reason I'm submitting this PR is because this field was being provided to a bot instance, and was failing deserialization, so, no, that note in the docs is wrong.

Given the conflicting info, it definitely sounds appropriate to keep it as `Optional<T>`, and I'll add a note regarding the conflicts.